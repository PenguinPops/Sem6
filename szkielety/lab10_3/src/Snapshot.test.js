import Snapshot from './Snapshot';
import renderer from 'react-test-renderer';

it('renderuje siÄ™ poprawnie', () => {
    const tree = renderer.create(<Snapshot />).toJSON();
    expect(tree).toMatchSnapshot();
})