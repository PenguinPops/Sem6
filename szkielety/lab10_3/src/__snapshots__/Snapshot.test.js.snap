// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderuje siÄ™ poprawnie 1`] = `null`;
